src/test/java/io/vertx/starter/service/GeneratorTest.java

```diff
@@ -292,7 +293,7 @@ private void verifyMavenFiles() {
 
   private void verifyMavenOutputFiles() throws IOException {
     try (Stream<Path> pathStream = Files.walk(workdir)) {
-      Optional<Path> testResult = pathStream.filter(p -> p.toString().endsWith("TestMainVerticle.txt")).findFirst();
+      Optional<Path> testResult = pathStream.filter(p -> p.toString().endsWith("TestMainVerticle.xml")).findFirst();
```

Comment:
@tsegismont ScalaTest does not generate `*.txt` files, except for the whole suite. To fix this, I had to change this predicate here, so it looks for the `xml` test report instead. I think this is also more consistent with `verifyGradleOutputFiles()`.

Reply:
@tsegismont just in case you missed it... This PR is ready to be reviewed now. 🤞 😄